komga-webui/src/locales/en.json

```diff
@@ -51,24 +51,24 @@
     "button_scan": "Scan",
     "button_select_series": "Select Series",
     "field_import_path": "Import from folder",
-    "info_part1": "This screen lets you import files that are outside your existing libraries. You can only import files into existing Series, in which case Komga will move or copy the files into the directory of the chosen Series.",
-    "info_part2": "If you choose a number for a book, and a book already exists with that number, then you will be able to compare the 2 books. If you decide to import the book, Komga will upgrade the existing book with the new one, effectively replacing the old file with the new.",
+    "info_part1": "Here you can import files into a selected series of your existing libraries.",
+    "info_part2": "Choose a number for a book and compare the two if it exists. Importing the book upgrades the entry and replaces the old file.",
     "no_files_found": "No files found",
     "notification": {
       "go_to_book": "Go to book",
-      "import_failure": "Failed to import book: {file}",
```

Comment:
Same "Could not" as above, only now more consistent.

Reply:
"Could not" and "failed" is widely different to me. The first one doesn't really explain what was the issue, the second one clearly involves some unexpected behaviour.

> Errors can be labeled on Weblate.

What do you mean?

Reply:
https://hosted.weblate.org/projects/remmina/remmina/nb_NO/ has "error messages"
I suppose it depends on whether software failing to do something is to be expected or considered anything other than a failure.

Reply:
Ok, Weblate has labels. But what is your point with that ?

Reply:
I am mentioning it so that errors can be tagged. This makes it easier for translators to do, and consistency improves.

Reply:
the problem i have with `{file}` being in the middle is that this string can get very long, as it denotes a full file path. By doing so, we wouldn't read the end of the sentence easily.